src/frontend/packages/store/src/reducers/pagination-reducer/pagination-reducer.types.ts

```diff
@@ -1,5 +1,5 @@
 
 
 export const resultPerPageParam = 'results-per-page';
-export const resultPerPageParamDefault = 5;
+export const resultPerPageParamDefault = 9;
```

Comment:
This should be kept at 5 I think. If you need it different you can supply `'results-per-page': 9` in your action's initialParams (this is a bit odd I know, going forward the stratos team need to fix this). There's also the `pageSizeOptions` array in `IListConfig` that is an array of page sizes.. the first one will automatically be picked.